### DISCUSSION TITLE
Can v0.1.0 already run the games shown in the screenshots?

### DISCUSSION THREAD

**Discussion by nyannoying1337** [InoriRus/Kyty#14] [Q&A] [answered]
I was wondering if v0.1.0 can already run games or do we need some extra things to make it work? so far i only get `"=== Unpatched function!!! ===
[9]     bzQExy189ZI[libc_v1][libc_v1.1][Func]
 in C:/projects/kyty/source/emulator/src/RuntimeLinker.cpp:505"` when trying to run Worms

**Comment by InoriRus** [comment, marked as answer]
Here is an example of configuration:

![image](https://user-images.githubusercontent.com/7149418/169696503-7b3e7a7e-25e3-449f-b514-53a6cc6b1ac8.png)

**Reply by Gaming1992** [reply]
can you tell in detail, after configuring like in the picture, what to do so the game can launch, I got the same problem, got an error in cmd like this?

**Reply by InoriRus** [reply]
What game did you try to launch?

**Reply by Gaming1992** [reply]
Cursed castila

**Reply by InoriRus** [reply]
I'll check it

**Reply by InoriRus** [reply]
It is possible to boot Cursed castila to main menu but then game crashes (shader recompiler fails):

![slice1](https://user-images.githubusercontent.com/7149418/170485074-c518df8b-991d-4074-8de6-742a7053f704.png)

**Reply by InoriRus** [reply]
Please show your configuration

**Comment by Reita111** [comment]
**Удаче в доработки эмулятора ,и новых игр ),увидел в дтв пост**
[https://dtf.ru/games/1205445-rossiyskiy-razrabotchik-pokazal-rannyuyu-versiyu-svoego-emulyatora-ps4-na-pk-on-vpervye-smog-zapustit-na-nem-igry?ref=recommended](url)
